server/src/com/thoughtworks/go/server/controller/JobController.java

```diff
@@ -192,6 +196,15 @@ public ModelAndView handleRequest(@RequestParam(value = "pipelineName") String p
         return jsonFound(json).respond(response);
     }
 
+    private AgentConfig agentConfig(String agentUuid) {
+        if (isBlank(agentUuid)) {
+            return null;
+        }
+
+        final AgentInstance agentInstance = agentService.findAgent(agentUuid);
+        return agentInstance == null ? null : agentInstance.agentConfig();
```

Comment:
What I meant is `agentInstance == null` will always be false since `agentService.findAgent(agentUuid)` will never return null.